challenge2Fizzbuzz.js

```diff
@@ -0,0 +1,13 @@
+function FizzBuzz(a, b) {

```

Comment:
This is duplicated alot, We need to talk about how you can use version control to mitigate this